api/test/unit/controllers/test_search_controller.py

```diff
@@ -30,17 +31,24 @@
 pytestmark = pytest.mark.django_db
 
 
-@pytest.fixture()
-def cache_setter():
-    keys = []
+cache_availability_params = pytest.mark.parametrize(
```

Comment:
Do you mind adding a comment here about what this particular fixture is for and what it's checking? Just for easier context when viewing the code!

Reply:
Added comment in 5622777.